.cirrus.yml

```diff
@@ -126,7 +126,7 @@ non_standard_toolchains_task:
     # build against libstatgrab, should always pass
     - env:
         LABEL: statgrab
-      allow_failures: false
+      allow_failures: true
```

Comment:
I would consider adding a TODO here, or reference this elsewhere for action (e.g. in an update to the container images in collectd/ci-docker).

However, this is mainly because I don't like making workarounds without documenting them somewhere